setup.py

```diff
@@ -15,7 +15,7 @@
     author="Gabriel Tseng",
     author_email="gabrieltseng95@gmail.com",
     url="https://github.com/nasaharvest/cropharvest",
-    version="0.2.0",
+    version="0.3.0",
```

Comment:
I reasoned that the inference class update necessitates a whole minor version update